spec/client_spec.rb

```diff
@@ -175,6 +175,16 @@ def wait_global_key_missing(key)
     @index.delete_index rescue "not fatal"
   end
 
+  it "should tell if index exists" do
+    @index.add_object!({ :name => "John Doe", :email => "john@doe.org" }, "1")
+    expect(@index.exists).to be true
+  end
+
+  it "should tell if index does not exist" do
+    index = Algolia::Index.new('test')
```

Comment:
You may want to use a special name here, or delete the current index. The possibilities of having a `test` name on the test cluster are high.